js/ui/codeView.js

```diff
@@ -453,6 +456,44 @@ var CodingSession = new Lang.Class({
         return this._toolbox;
     },
 
+    setGrabbed: function(grabbed) {
+        if (this._button)
+            this._button.opacity = grabbed ? 0 : 255;
+
+        if (this.toolbox && this._state == STATE_APP)
+            this.toolbox.opacity = grabbed ? 0 : 255;
+    },
+
+    _attachAppToToolbox: function() {
+        this._appParent = this.app.get_parent();
+        this._appParent.remove_child(this.app);
+        this.toolbox.insert_child_at_index(this.app, 0);
+        this.app.set_position(0, 0);
+        this._appPositionId = this.app.connect('notify::position', (actor) => {
+            const [x, y] = actor.get_position();
+            if (x || y)
+                actor.set_position(0, 0);
+        });
+    },
+
+    _detachAppFromToolbox: function() {
+        if (!this._appParent || !this.toolbox) {
+            return;
+        }
```

Comment:
Assuming the convention is to not use curly brackets in one-liner if statements (like you have in the beginning of this patch), then this should be removed.